src/geouned/GEOUNED/core.py

```diff
@@ -681,8 +681,8 @@ def start(self):
                 if c.Definition.level == 0 or c.IsEnclosure:
                     continue
                 logger.info(f"simplify cell {c.__id__}")
-                Box = UF.get_box(c)
-                CT = build_c_table_from_solids(Box, (c.Surfaces, Surfs), option="full")
+                Box = UF.get_box(c,self.options)
+                CT = build_c_table_from_solids(Box, (c.Surfaces, Surfs), option="full",options=self.options)
```

Comment:
```suggestion
                CT = build_c_table_from_solids(Box, (c.Surfaces, Surfs),options=self.options)
```
here I would prefer to pass only the ```self.options``` class as argument.

Also to avoid mistake the ```option``` attribute name of the ```self.options``` class should be changed from ```option``` to ```simplification_mode```

Reply:
Unless I'm missing something, it doesn't look like just self.options will work. I believe simplify is contained in self.settings.simplify.

Reply:
Yes, you are right. The quick review made me make a mistake.